apps/oc_chef_wm/priv/dispatch.conf

```diff
@@ -131,6 +131,8 @@
 
 %% Cookbook artifact endpoints
 {["organizations", organization_id, "cookbook_artifacts", name, identifier], oc_chef_wm_named_cookbook_artifact_version, []}.
+{["organizations", organization_id, "cookbook_artifacts", name], oc_chef_wm_cookbook_artifacts, [{resource_args, single_artifact}]}.
```

Comment:
We'll usually handle this in named_X, with 'identifier' being the special case within named_X.  I was wondering why the difference in this case?